outscale/resource_outscale_access_key_test.go

```diff
@@ -117,6 +117,45 @@ func TestAccOutscaleAccessKey_updatedToActivedKey(t *testing.T) {
 	})
 }
 
+func TestAccOutscaleAccessKey_updatedExpirationDate(t *testing.T) {
+	resourceName := "outscale_access_key.outscale_access_key"
+
+	expirDate := "2025-01-04"
```

Comment:
In 3 years, we might have a problem :) We could use [Time](https://pkg.go.dev/time#Time.AddDate) module to have something dynamic